packages/react-instantsearch-hooks-dom/src/ui/Hits.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+
+import type { Hit } from 'instantsearch.js';
```

Comment:
Don't we order imports?

Reply:
they are imported, this is a type import which comes last (eslint deals with it)